assets/src/app/routes.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { Route, RouteComponentProps, Switch, Redirect } from 'react-router-dom';
 import { accessibleRouteChangeHandler } from '@app/utils/utils';
 import { ATODocuments } from '@app/ato/Documents/Overview';
-import { ATOTrainingPlan, ATOVulnerabilityManagementPlan, ComponentFedRAMPTemplates } from '@app/ato/Documents/Documents';
+import { ATOTrainingPlan, ATOVulnerabilityManagementPlan, ComponentSSPTemplates } from '@app/ato/Documents/Documents';
```

Comment:
Perhaps, unused import?

Reply:
Yes - but let's create another issue. I plan to reformat and clean up unused imports this Q.